zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -319,7 +319,7 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
         "404":
-          description: "Not found"
+          description: "Bad request"
```

Comment:
❌ We must not override this back to the previous values. The 404 code is clearly a "Not found".